prisma/devSetup.ts

```diff
@@ -0,0 +1,211 @@
+/* eslint-disable func-names */
+/* eslint-disable no-console */
+import { PrismaClient } from '@prisma/client';
+import loadCSVFromFile from './loadCsv';
+
+const seedHealthcareProfessionals = async function (prisma: PrismaClient, verbose = false) {
+  const firstEn = 0;
+  const middleEn = 1;
+  const lastEn = 2;
+  const firstJa = 3;
+  const middleJa = 4;
+  const lastJa = 5;
+  const degreeCol = 6;
+  const specialtyCol = 7;
+  const spokenLangCol = 8;
+
+  const devData:string[][] = loadCSVFromFile('./prisma/seedData/devHealthcareProfessionals.csv');
+
+  devData.forEach(async (row: string[], index: number) => {
+    // generate some unique ids using even/odd strategy
+    const enID = 2 * index;
```

Comment:
future issue: before we go to production, it would be a good idea to use non-predictable ids for security purposes.

Reply:
Ahh, good thought. Do you think this matters for non-PII data like degrees, specialities, etc?

Reply:
for privacy, no.....we should be good. 👍  Foreign keys to that data will be with the PII data that should be encrypted like healthcare professionals, facilities, etc..  There's no risk/merit of black hat usage.